src/main/java/Wolox/training/models/Book.java

```diff
@@ -0,0 +1,134 @@
+package Wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import org.springframework.data.annotation.PersistenceConstructor;
+
+@Entity
+public class Book {
+
+    @PersistenceConstructor
+    public Book(Integer id, String genre, String author, String image, String title,
+            String subtitle,
+            String publisher, String year, Integer pages, String isbn) {
+        this.id = id;
+        this.genre = genre;
+        this.author = author;
+        this.image = image;
+        this.title = title;
+        this.subtitle = subtitle;
+        this.publisher = publisher;
+        this.year = year;
+        this.pages = pages;
+        this.isbn = isbn;
+    }
+
+    public Book() {
+        super();
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
We use SEQUENCE instead of AUTO, because postgre we use AUTO creates a single autoincremental id for all tables